arch/arm/core/cpu_idle.S

```diff
@@ -120,7 +124,8 @@ SECTION_FUNC(TEXT, k_cpu_idle)
 	mov lr, r0
 #endif
 
-#if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
+#if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE) \
```

Comment:
This, here (and right below on line 180/185)  look a bit weird; ARMV7_R is "grouped" together with ARMV6_M, instead of ARMV7_M. 

If this is indeed correct, and needs to be as is, it will confuse the reader.
Any suggestion how to address this, @galak ?